crates/hardware/src/lib.rs

```diff
@@ -43,6 +44,11 @@ pub trait PathsInterface {
     fn get_paths(&self) -> Paths;
 }
 
+pub trait RecordingInterface {
+    fn get_recording(&self) -> bool;
+    fn set_recording(&self, enable: bool);
```

Comment:
As discussed. Please reiterate on the naming here ;)

Reply:
Renamed to

- "should record"
- "set whether to record"

Feel free to propose something :wink:

Reply:
To explain my motivation in keeping it for now and trying to shortcut the discussion: I think we should keep in mind the intended maturity level. The long-term plan is (as written in a comment earlier) that the `framework.json`, Communication, and `HardwareInterface` will provide whether they want to record, in which frequency, and/or whether they want to record a burst for a given time duration. This will all be some state that is written once in a cycle or in a lower frequency. The simple boolean that currently exists will be replaced by that. Therefore, I think we should not overfit to it now. In addition, I would vote against making the robotics domain longer due to calling two functions instead of one. The idea of "record this cycle" changes the API and requires additional state and code to reset the state, while we are not yet sure whether this additional code can survive under the planned ideas.